app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -1,10 +1,16 @@
-@import 'custom-variables';
+@import 'variables';
 @import 'bootstrap/scss/bootstrap';
 
 @import 'helpers';
 @import 'rooms';
 
+html,
 body {
-  background-color: whitesmoke;
+  height: 100%;
```

Comment:
What is the use of `height: 100%` here?

Reply:
Make the body take the full height of the browser, so that we can later on make the background take the full height as well

Reply:
Alright, thanks thought that by default the body occupies 100% of the page.